src/authentication/register.js

```diff
@@ -21,14 +22,15 @@ exports.register = ( request, response, next ) => {
         return response.status( 422 ).send({ error: ERROR_MESSAGE })
       }
 
-      const user = new User({ email, password })
+      const user = new User({
+        name, email, password, phone_number, newsletter_subscribed, company
+      })
 
       user.save(( err, user ) => {
         if( err ) {
           return next( err )
         }
-
-        return response.status( 201 ).json( tokenInfo( user ))
+        response.redirect('http://localhost:3000/dashboard')
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```